lua/terrortown/lang/en.lua

```diff
@@ -1792,8 +1792,8 @@ L.equipmenteditor_name_kind = "Equipment Slot"
 L.equipmenteditor_desc_kind = "The inventory slot the equipment will occupy."
 
 L.slot_weapon_melee = "Melee Slot"
-L.slot_weapon_pistol = "Pistol Slot"
-L.slot_weapon_heavy = "Heavy Slot"
+L.slot_weapon_pistol = "Secondary Slot"
+L.slot_weapon_heavy = "Primary Slot"
```

Comment:
while this is probably a good change, I wonder if this is confusing

Reply:
It's kinda confusing, since if you look at `label_max_primary_slots` and `label_max_secondary_slots` they are spelled this way, so I just wanted them to be consistent.